### DISCUSSION TITLE
Google Chat integration

### DISCUSSION THREAD

**Discussion by deanegan** [ptarmiganlabs/butler-sos#425] [General]
Hello,

I am aware that ButlerSoS can integrate easily with Slack but is it the same with Google Chat?

Many Thanks,
Dean

**Comment by mountaindude** [comment]
Hmm, Butler SOS does not have any Slack features, I believe..

Are you possibly thinking of the original Butler tool ([butler.ptarmiganlabs.com](https://butler.ptarmiganlabs.com))?
That tool can monitor task reload failures and send alerts to Slack and Teams. But not to Google chat, correct.

No immediate plan to add that, but I can't see it being very difficult so if you're up for some coding - feel free to take a stab at it!
